vector-im/element-android
# Voice Broadcast - Update last message in the room list

## Type of change


## Content

Display "Live broadcast" in the room list if a VB is live
Show a notice in the room list and in the timeline when a voice broadcast is ended

Improve RoomSummary.latestPreviewableEvent computation in the sdk:
- Event types with a reference relationship are not filtered anymore (was done to filter poll answers in encrypted events)
- Filter the encrypted events by checking the clear type instead

Note: For the moment, voice messages related to a VB are not filtered out on the sdk side, if the computed `latestPreviewableEvent` is a VB message, the application will just display a blank instead.

## Motivation and context

Continue #7127 

Design request:
<img width="239" alt="image" src="https://user-images.githubusercontent.com/11990514/205863039-d060a55c-df51-4055-8274-c65ffa9f9321.png">

## Screenshots / GIFs

TODO



## Tests

- Start a VB from device A
- On device B, verify that the last message in the related room is "Live Broadcast" (even if the VB is paused/resumed)
- Verify that new messages or typing events are not shown in the room list while the VB is live
- Verify that a notice is display in the room list (last message) and in the timeline when the VB is stopped
- Verify that voice messages related to the VB are not shown in the room list (if a voice message is catch after the stopped event, it will be blank instead)
- Verify that typing events and new messages are shown again

## Tested devices

- OS version(s):

## Checklist
